ghcup-0.0.7.yaml

```diff
@@ -4407,6 +4405,77 @@ ghcupDownloads:
             unknown_versioning:
               dlUri: https://downloads.haskell.org/~ghcup/unofficial-bindists/cabal/3.10.1.0/cabal-install-3.10.1.0-armv7-linux-deb10.tar.xz
               dlHash: 79869dd72cb0a6f90f2e1e0e57af4ea6ee09b041550abc09252d1cf309b4f96e
+    3.10.2.0:
+      viPostInstall: "Metadata validation is currently broken on Windows, please see https://github.com/haskell/ghcup-metadata/issues/142."
```

Comment:
This is a confusing message. The end user doesn't really know what metadata validation is, nor do they care about ghcup CI. And it's not metadata validation that's broken... it works perfectly fine. `cabal run` is broken. So the message should state that `cabal run` is broken on windows and link to the issue.

Reply:
Ah sorry, I had misunderstood your message